### PR TITLE
SOC-5347: Prevent unauthenticated users to access path /rest/jcr/repo…

### PR DESCRIPTION
**Pull request by kmenzli**
…sitory/social/production

### CODE REVIEW

File: extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml [hunk 1]
```diff
@@ -198,20 +198,20 @@
       </component-plugin>
   </external-component-plugins>
 
-  <external-component-plugins>
```
Comment (thomasdelhomenie) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml:201]: Is this change really related to this issue ?

Comment (kmenzli) [inline extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/component-plugins-configuration.xml:201]: I'll check again, may a typo on my PR
